src/app/shared/interfaces/error.interface.ts

```diff
@@ -0,0 +1,27 @@
+export interface ErrorInside {
+    code: number;
+    message: string;
+
+}
+export interface Error {
+    error: ErrorInside;
+}
+
+export interface IHeaders {
+    lazyInit: any;
+    lazyUpdate: any;
+    normalizedNames: any;
+}
+
+export interface IError {
```

Comment:
I would move this interface to the line 10 to have all the Error interfaces together

Reply:
Done!